docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -431,15 +433,15 @@ for i, t in enumerate(df_retweet['tweet_id']):
 
 When running this loop, you are likely to see this:
 
-```python
+```{ .yaml .no-copy }
 rate limit exceeded: sleeping 567.0111730098724 secs
 ```
 
 This is because we are making a lot of calls to the Twitter API. The scraping will proceed after the sleep time is over.
 
 Then we combine with tweets dataframe, throw away duplicated tweets and sort by `created_at` date.
 
-```python
+```bash linenums="1" title="python"
```

Comment:
python

Reply:
I think you're looking at outdated file but I changed all python to py